### PR TITLE
Include Microchip RN4870/1 UUIDs

### PR DESCRIPTION
**Pull request by furbrain**
Hi Kai,
I'm working on a project using the Microchip RN4870 - your app on the play store works really well - but I needed to setup custom UUIDs. I've found this code and wonder if you would accept this pull request to 
include the microchip UUIDs. I've tested it on my setup and all works very nicely.

Yours,

Phil Underwood

### CODE REVIEW

**Comment by kai-morich** [comment]
in microchip documentation I found differnt characteristics for read & write. you use the same. does it work in both directions?

**Comment by furbrain** [comment]
Yes, it does work. I have tested both send and receive. From the documents (my italics):

> The Transparent UART TX Characteristic is used for data transmission by the Server
or the Client. Once the Client Characteristic Configuration Descriptor (CCCD) of
Transparent UART TX Characteristic is enabled, the Server sends data to the Client
using the Notify property. _The Client can also send data to the Server using the
Write/Write without response properties._

**Comment by kai-morich** [comment]
got it. wasn't obvious when just looking at the characteristics names, but further down it was explained in RN... documentation.

File: app/src/main/java/de/kai_morich/simple_bluetooth_le_terminal/SerialSocket.java [hunk 1]
```diff
@@ -32,6 +32,8 @@
     private static final UUID BLUETOOTH_LE_NRF_SERVICE    = UUID.fromString("6e400001-b5a3-f393-e0a9-e50e24dcca9e");
     private static final UUID BLUETOOTH_LE_NRF_CHAR_RW2   = UUID.fromString("6e400002-b5a3-f393-e0a9-e50e24dcca9e"); // read on microbit, write on adafruit
     private static final UUID BLUETOOTH_LE_NRF_CHAR_RW3   = UUID.fromString("6e400003-b5a3-f393-e0a9-e50e24dcca9e");
+    private static final UUID BLUETOOTH_LE_RN4870_SERVICE = UUID.fromString("49535343-FE7D-4AE5-8FA9-9FAFD205E455");
+    private static final UUID BLUETOOTH_LE_RN4870_RW      = UUID.fromString("49535343-1E4D-4BD9-BA61-23C647249616");
```
Comment (kai-morich) [inline app/src/main/java/de/kai_morich/simple_bluetooth_le_terminal/SerialSocket.java]: for consistency please change name to ..._CHAR_RW

Comment (furbrain) [reply app/src/main/java/de/kai_morich/simple_bluetooth_le_terminal/SerialSocket.java]: Ok, done

**Pull request merged**